### ISSUE TITLE
java.lang.ClassCastException console spam

### ISSUE THREAD

**Issue by OoLunar** [RelativityMC/raknetify#17]
**Describe the bug**
After the UDP server starts, Raknet spams console with this when a player joins.

**To Reproduce**
1. Install Raknet
2. Startup the server
3. Let the player join and run around
4. Check console

**Expected behavior**
For my console to be clear without exceptions.

**Screenshots**
N/A

**Runtime info (please complete the following information):**
 - OS: 
   - Host: Void Linux
   - Docker: `itzg/minecraft-server` (Ubuntu, Hotspot Java 17)
 - Minecraft version: 1.19.2
 - Mod version: [Build 106](https://ci.codemc.io/job/RelativityMC/job/raknetify/job/master/106/) from CodeMC
 - Mod branch: N/A

**Crash reports / logs**
```
[19:26:28] [Raknetify NIO #0/WARN]: An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
java.lang.ClassCastException: class io.netty.channel.socket.DatagramPacket cannot be cast to class io.netty.channel.Channel (io.netty.channel.socket.DatagramPacket and io.netty.channel.Channel are in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @97e1986)
	at org.geysermc.floodgate.inject.fabric.FabricInjector$1.channelRead(FabricInjector.java:25) ~[floodgate-fabric.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [Geyser-Fabric.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [Geyser-Fabric.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [Geyser-Fabric.jar:?]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) [Geyser-Fabric.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [Geyser-Fabric.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [Geyser-Fabric.jar:?]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) [Geyser-Fabric.jar:?]
	at network.ycc.raknet.channel.DatagramChannelProxy$ListenerInboundProxy.channelRead(DatagramChannelProxy.java:371) [raknetify-fabric-0.1.0+alpha.5.21-all.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [Geyser-Fabric.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [Geyser-Fabric.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [Geyser-Fabric.jar:?]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) [Geyser-Fabric.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [Geyser-Fabric.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [Geyser-Fabric.jar:?]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) [Geyser-Fabric.jar:?]
	at io.netty.channel.nio.AbstractNioMessageChannel$NioMessageUnsafe.read(AbstractNioMessageChannel.java:97) [Geyser-Fabric.jar:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:719) [Geyser-Fabric.jar:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655) [Geyser-Fabric.jar:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581) [Geyser-Fabric.jar:?]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493) [Geyser-Fabric.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986) [Geyser-Fabric.jar:?]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [Geyser-Fabric.jar:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [Geyser-Fabric.jar:?]
	at java.lang.Thread.run(Unknown Source) [?:?]
```

**Other mods**
From Console: https://code.forsaken-borders.net/dogaxenato.css
From `ls *.jar | cat`: https://code.forsaken-borders.net/ezocuhadib.css

**Checklist**
 - [x] I am using the official version of the mod. 
 - [x] I tried the latest development version but the issue persists. 
 - [x] I searched for similar open issues and could not find an existing bug report on this. 

**Additional context**
The server has Krypton, Geyser and Floodgate installed.
All Java players have the Krypton mod client-side.
Raknet Java players have reported no issues.
Normal Java players have reported no issues.
1 Bedrock player connecting through Geyser and Floodgate has reported that they do not receive chat messages while the other Bedrock player has sent screenshots of them receiving chat messages.

**Comment by OoLunar** [comment]
After looking at the `latest.log` file from issue #4, this appears to be a Floodgate incompatibility.

**Comment by toinouH** [comment]
@OoLunar You need one port for raknetify and one for Geyser+Floodgate ^^

**Comment by OoLunar** [comment]
> @OoLunar You need one port for raknetify and one for Geyser+Floodgate ^^

I am aware, they are on two separate UDP ports

**Comment by ishland** [comment]
Fixed in #4
